add blogpost - SGID Then and Now (#1119)
agrc/gis.utah.gov: _posts/2019-06-11-sgid-then-and-now.md

---
title: The SGID - Then and Now
author:
  display_name: Greg Bunce
  email: gbunce@utah.gov
date: 2019-06-11 10:07:46
categories:
  - Featured
tags:
  - sgid
  - data
  - education
  - gis history
---

Access to high-quality GIS data is a privilege, and the fact that it's open and free is nothing short of amazing.  Yes, I’m talking about Utah's State Geographic Information Database (SGID).  From a local perspective, we’re all accustomed to it being there.  But really, this concept of readily-available, mature, open data is somewhat unique.  Over the years, many factors have played into the SGID’s development and current maturity.  Likewise, It takes ongoing maintenance and planning to ensure its continued success.

This post explores some of the early milestones (‘highlights’) of the SGID, the current state of the SGID with usage stats, and finally the ‘where-are-we going-next’ - with a [survey](https://www.surveymonkey.com/r/3P58K98) requesting user-input!

> ### `Early History of the SGID`

Reviewer: ```suggestion
### Early History of the SGID
```

In 1980, ESRI was hired as a consultant to assist with Utah’s GIS implementation plan. The following year the state signed a contract with ESRI to purchase hardware, software, and training.  The software they purchased pre-dated ARC/INFO, which was released a year later in 1982 under the banner [“the first commercial GIS”](https://www.esri.com/~/media/Files/Pdfs/about-esri/esri-history-up-close).  But, it was with this equipment that the state began creating Utah’s first statewide GIS database.

In 1982, the Automated Geographic Reference, or AGR, was established with the goal to catalog and maintain this data.  AGR entered the modern era in 1984 with the purchase of ARC/INFO on a PRIME minicomputer.

In 1989, AGRC was created (replacing AGR).  That change brought new priorities to the agency and at the top of the list was the completion of the SGID.  AGRC entered into a contract with the Governor’s Office of Planning and Budget for $80,000 to assist in the effort.  As a result, the first SGID infrastructure was completed in 1991.

> The SGID was further solidified in 1991 with the passage of [Senate Bill 21](https://le.utah.gov/xcode/Title63F/Chapter1/63F-1-S507.html).  Passing unanimously, this bill created the AGRC and the SGID in state statute and eliminated many uncertainties such as future funding for the database and AGRC’s responsibilities.

Reviewer: This creates a bug. There should be no space between the `] (`

> The SGID was further solidified in 1991 with the passage of [Senate Bill 21](https://le.utah.gov/xcode/Title63F/Chapter1/63F-1-S507.html).  Passing unanimously, this bill created the AGRC and the SGID in state statute and eliminated many uncertainties such as future funding for the database and AGRC’s responsibilities.

Author: the url does not want to open.  i'll keep it as is

The SGID has been in continuous operation since it’s completion in 1991.  Since then, It has seen numerous software and hardware configurations.  

Here are a few of the ‘highlights’ that have brought us to where we are now:

- **SGID User's Guide - 7th Edition (1994)**
  - This user guide was distributed in a three-ring binder. Data was then requested in the following available formats: ARC/INFO Coverages, USGS DLG, MOSS Export, and AutoCAD DXF. This guide also made note of an 'AGRC home page on the world wide web' which was to be debuting in November 1994.

![1994 SGID User Guide Binder]({{ "/images/SGID_1994_binders_cover_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}

> - **The SGID on CD-ROM (1995)**

Author: the cd-rom does not specify a version, so i didn't want to assume anything. i think it's fine to leave it out.

>   - In 1995, AGRC, ESRI, and Utah State University combined efforts and manufactured a CD-ROM containing SGID data and a built-in data viewer. To commemorate the milestone, we have made the available the [original CD-ROM jacket](https://drive.google.com/open?id=17z_AYz68fa2ogToRIIBgX9rTprmA-U1b) as well as the [unaltered CD-ROM contents](https://drive.google.com/open?id=17h13sLMTe3k_g_WnhbphAI-dLPwp5odo).

Reviewer: ```suggestion
  - In 1995, AGRC, ESRI, and Utah State University combined efforts and manufactured a CD-ROM containing SGID data and a built-in data viewer. To commemorate the milestone, we have made available the [original CD-ROM jacket](https://drive.google.com/open?id=17z_AYz68fa2ogToRIIBgX9rTprmA-U1b) as well as the [unaltered CD-ROM contents](https://drive.google.com/open?id=17h13sLMTe3k_g_WnhbphAI-dLPwp5odo).
```

![1995 SGID CD-ROM photo]({{ "/images/SGID_1995_cover_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}

- **SGID User’s Guide -10th Edition (1999)**
  - This version of the user's guide was also distributed in a three-ring binder, and talked about the AGRC home page as the place where you can get direct access to the data, as well as view the FGDC metadata. It also mentions an FTP site. It notes that this version will be last printed user guide and that future guides will be available on the website for download in PDF format. Users could also request the data in the following formats: CD, 3.5 diskettes, 8mm tape, zip disc, or Coverages (which it mentions are the most popular).

![1999 SGID User Guide Binder]({{ "/images/SGID_1999_binders_cover_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}

> - **SGID meets the World Wide Web (the early 2000s)**

Author: i was not able to find a version on this either.  maybe the versions were just on the printed three-ring binder guides (?).

  - By the early 2000s the SGID user's guide, data, and data viewer were all solely available over the Web. AGRC was using ArcSDE 8.2 to maintain the data and ArcIMS for the interactive mapping website. Users could also use a separate ArcIMS site to create their own exportable map using a [custom map authoring tool]({{ "/images/SGID_early2000s_create_custom_map.gif" | prepend: site.baseurl }}).

[![SGID Early 2000s]({{ "/images/SGID_early2000s_sm.png" | prepend: site.baseurl }}){: .flex .flex--center}]({{ "/images/SGID_early2000s.png" | prepend: site.baseurl }})

> ### `Where is the SGID Now?`

Reviewer: ```suggestion
### Where is the SGID Now?
```

{: .text-left}

Technology and the use of GIS have come a long way since the inaugural SGID of 1991.   What hasn't changed, though, is the notion of a centralized repository for searching and fetching statewide GIS data.  Sure, file formats and how we access data has changed, but the need for open, readily-accessible GIS data remains the same.

> The current makeup of the SGID is distributed, by design.  Internally, much of the framework data is managed on a SQL ArcSDE Database.  From there, data layers are distributed accordingly.

Author: i say "sequel", but on this one, it depends on who you are talking, too.  I took a "sequel" class and the instructor went out his way to say "S-Q-L" all week.

> The [sgid index]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and user guide are located on [AGRC’s website]({{ "/data/" | prepend: site.baseurl }}).  Here you will find descriptions of the data, update dates, data downloads, as well as links to other places where you can access the data.  It is recommended that you begin your data search there.

Reviewer 2: ```suggestion
The [SGID index]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and current user guide are located on [the SGID page of AGRC’s website]({{ "/data/" | prepend: site.baseurl }}), where you will also find descriptions of the data, update dates, data downloads, and links to other places where you can access the data. If you're looking for data, that is the best place to start. 
```

> The [index catalog]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and user guide are located on the [AGRC’s website]({{ "/data/" | prepend: site.baseurl }}).  Here you will find descriptions of the data, update dates, data downloads, as well as links to other places where you can access the data.  It is recommended that you begin your data search there.

Reviewer: ```suggestion
The [sgid index]({{ "/data/sgid-index/" | prepend: site.baseurl }}) and user guide are located on the [AGRC’s website]({{ "/data/" | prepend: site.baseurl }}).  Here you will find descriptions of the data, update dates, data downloads, as well as links to other places where you can access the data.  It is recommended that you begin your data search there.
```

![SGID 2019 User Guide]({{ "/images/SGID_2019_webpage_data.png" | prepend: site.baseurl }}){: .flex .flex--center}

> In order to meet the diverse needs of our users and applications, SGID data is served on a variety of platforms in a variety of formats, including direct database access, downloadable data, and API web services.

Reviewer: what about open/data arcgis online?

Author: good point.  thanks

> Here are some current SGID stats, showing top accessed data layers by platform:

Reviewer 2: ```suggestion
Here are some current SGID stats showing top accessed data layers by platform:
```

> - **[AGRC Website](https://gis.utah.gov/data)** - Page-views from the past 12 months - rounded to the nearest one-hundred

Reviewer 2: ```suggestion
- **[AGRC Website](https://gis.utah.gov/data)** - Page views from the past 12 months (rounded to the nearest one-hundred)
```

>   - Parcels - (30,900 page-views)

Reviewer 2: ```suggestion
  - Parcels - 30,900
```

>   - Elevation and Terrain - (20,400)

Reviewer 2: ```suggestion
  - Elevation and Terrain - 20,400
```

>   - Aerial Photography - (20,100)

Reviewer 2: ```suggestion
  - Aerial Photography - 20,100
```

>   - Transportation - (18,200)

Reviewer 2: ```suggestion
  - Transportation - 18,200
```

>   - USGS Scanned Topo Maps - (15,900)

Reviewer 2: ```suggestion
  - USGS Scanned Topo Maps - 15,900
```

>   - Boundaries - Landownership - (15,600)

Reviewer 2: ```suggestion
  - Boundaries - Landownership - 15,600
```

>   - Boundaries - City, County, and State - (15,300)

Reviewer 2: ```suggestion
  - Boundaries - City, County, and State - 15,300
```

>   - Oil and Gas - (8,000)

Reviewer 2: ```suggestion
  - Oil and Gas - 8,000
```

>   - Water - (7,600)

Reviewer 2: ```suggestion
  - Water - 7,600
```

>   - Basemap and Imagery - (5,300)

Reviewer 2: ```suggestion
  - Basemap and Imagery - 5,300
```

> - **[ArcGIS Online](https://utah.maps.arcgis.com)** - The most popular content from June 2013 to May 2019, rounded to the nearest one-thousand

Reviewer 2: ```suggestion
- **[ArcGIS Online](https://utah.maps.arcgis.com)** - The most popular content from June 2013 to May 2019, as measured in page views (rounded to the nearest one-thousand)
```

Reviewer: It's not just page views

  - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, UHP Dispatch Zones) - (67,000 views)

>   -  Roads (minor and major roads combined stats) - (64,000)

Reviewer: should this one say the measurement? requests?

Author: i took the term "most popular content" from the agol dashboard where I got the numbers.  It's a little vague to me as well, but I better stick to it b/c I don't know the exact metric.

>   - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, UHP Dispatch Zones) - (67,000 views)

Reviewer 2: ```suggestion
  - SBI Operations (which includes: Law Enforcement Boundaries, City Courts, County Courts, State District Courts, and UHP Dispatch Zones) - 67,000
```

>   -  Roads (minor and major roads combined stats) - (64,000)

Reviewer 2: ```suggestion
  -  Roads (minor and major roads combined stats) - 64,000
```

>   -  County Boundaries - (47,000)

Reviewer 2: ```suggestion
  -  County Boundaries - 47,000
```

>   -  Municipal Boundaries - (35,000)

Reviewer 2: ```suggestion
  -  Municipal Boundaries - 35,000
```

>   -  Parcels - (28,000)

Reviewer 2: ```suggestion
  -  Parcels - 28,000
```

>   -  Lakes - (26,000)

Reviewer 2: ```suggestion
  -  Lakes - 26,000
```

>   -  Address Points - (17,000)

Reviewer 2: ```suggestion
  -  Address Points - 17,000
```

>   -  Zip Codes - (17,000)

Reviewer 2: ```suggestion
  -  Zip Codes - 17,000
```

> - **SQL ArcSDE Database** - Desktop-user Views, from the past 12 months (approximately), rounded to the nearest one-hundred

Reviewer 2: ```suggestion
- **SQL ArcSDE Database** - Desktop-user views from the past 12 months (approximately) rounded to the nearest one-hundred
```

Reviewer 2: Are "desktop-user views" different from the page views referenced for the other data layers?...

>   - Parcels, all counties combined - (14,000 table-locks)

Reviewer 2: ```suggestion
  - Parcels, all counties combined - 14,000 table-locks
```

Reviewer 2: What are table-locks?...

>   - Counties - (8,000)

Reviewer 2: ```suggestion
  - Counties - 8,000
```

>   - Municipalities - (6,100)

Reviewer 2: ```suggestion
  - Municipalities - 6,100
```

>   - Landownership - (4,900)

Reviewer 2: ```suggestion
  - Landownership - 4,900
```

>   - Roads - (4,700)

Reviewer 2: ```suggestion
  - Roads - 4,700
```

>   - Sections - (4,000)

Reviewer 2: ```suggestion
  - Sections - 4,000
```

>   - Townships - (3,600)

Reviewer 2: ```suggestion
  - Townships - 3,600
```

>   - Quarter-Quarter Sections - (1,500)

Reviewer 2: ```suggestion
  - Quarter-Quarter Sections - 1,500
```

>   - Zip Codes - (2,100)

Reviewer 2: ```suggestion
  - Zip Codes - 2,100
```

>   - Address Points - (2,100)

Reviewer 2: ```suggestion
  - Address Points - 2,100
```

> - **[OpenData website](https://opendata.gis.utah.gov/)**

Reviewer 2: ```suggestion
- **[OpenData Website](https://opendata.gis.utah.gov/)**
```

  - Currently, averaging 48 item views a day (17,413 views over the past year)

> ### `Where is the SGID Going?`

Reviewer: ```suggestion
### Where is the SGID Going?
```

>   - Currently, averaging 48 item views a day (17,413 views over the past year)

Reviewer 2: ```suggestion
  - Currently averaging 48 item views a day (17,413 views over the past year)
```

> ### Where is the SGID Going?

Reviewer 2: ```suggestion
### Where Is the SGID Going?
```

{: .text-left}

> One of the challenges of working in technology is that over time file formats, and the way we access data, change.  This is similarly true for the SGID.  As a result, AGRC engages in regular discussions on where the SGID should be going.  

Reviewer 2: ```suggestion
One of the challenges of working in technology is that over time, file formats and the way we access data change. This is similarly true for the SGID. As a result, AGRC engages in regular discussions on where the SGID should be going.  
```

> Currently, we are working toward [trimming down the SQL ArcSDE enterprise database]({{site.baseurl}}{% post_url 2018-09-10-the-incredible-shrinking-sgid %}), which is available to those on the state network.  Moving forward, we will maintain only ‘live’ datasets in the SDE.  These would be datasets that receive regular modifications, such as address points, parcels, municipal boundaries, roads, etc. The idea is to use this database as our internal, editing, and staging database.

Reviewer 2: ```suggestion
Currently, we are working toward [trimming down the SQL ArcSDE enterprise database]({{site.baseurl}}{% post_url 2018-09-10-the-incredible-shrinking-sgid %}), which is available to users on the State network.  Moving forward, we will maintain only "live" datasets in the SDE. These would be datasets that receive regular modifications, such as address points, parcels, municipal boundaries, roads, etc. The idea is to use this database as our internal editing and staging database.
```

> All data layers being maintained in the SDE will also be available in [ArcGIS Online (AGOL)](https://utah.maps.arcgis.com) and [AGRC’s open data portal](https://opendata.gis.utah.gov/).  Here users can consume the web service or download the data.  Obsolete data will be written to object storage for retention and available on request.  

Reviewer 2: ```suggestion
All data layers being maintained in the SDE will also be available in [AGOL](https://utah.maps.arcgis.com) and [AGRC’s open data portal](https://opendata.gis.utah.gov/). Here users can consume the web service or download the data. Obsolete data will be written to object storage for retention and will be available on request.  
```

> In the near future, AGRC will also offer a public-facing, PostgreSQL cloud database where users can make direct connections from their desktop client.  This will bring new functionality as well as an additional access point for our end users.  We are excited about this offering so keep an eye on our [Twitter](https://twitter.com/maputah) or [newsletter](https://gis.utah.gov/news/#mc_embed_signup) for upcoming details.  

Reviewer: Are we getting ahead of ourselves here? Are we planning to test this first before committing to it or are we all in?

Author: we could say we're 'testing the option'.  though, i have an arcgis desktop connection to the postgres database that keaton built and a few of us have tested it. out a bit  it seems pretty solid.  I guess we just need the data quality checks and the update cycle ironed out.   what do you think, 'testing the option'?

Reviewer: that seems like safe wording

> In the near future, AGRC will also offer a public-facing, PostgreSQL cloud database where users can make direct connections from their desktop client.  This will bring new functionality as well as an additional access point for our end users.  We are excited about this offering so keep an eye on our [Twitter](https://twitter.com/maputah) or [newsletter](https://gis.utah.gov/news/#mc_embed_signup) for upcoming details.  

Reviewer: ```suggestion
In the near future, AGRC will also offer a public-facing, PostgreSQL cloud database where users can make direct connections from their desktop client.  This will bring new functionality as well as an additional access point for our end users.  We are excited about this offering so keep an eye on our [Twitter](https://twitter.com/maputah) or [newsletter]({{ "/news/#mc_embed_signup" | prepend: site.baseurl }}) for upcoming details.  
```

Ultimately, our goal is provided the most-current, industry-accepted tools that allow our users to access high-quality, standardized, statewide GIS data for Utah.  Additionally, we design or paths forward with a reasonable amount of backward capability.  

We are always curious to hear from our users on how you access the data, what formats you use, and how we can better get the data into your hands.  Please take a moment to fill out our one-question [poll](https://www.surveymonkey.com/r/3P58K98) on your preferred method(s) of accessing SGID data.  This helps us gauge where we should focus our future efforts.

> [![Take the Survey]({{ "/images/SGID_2019_take_the_poll.png" | prepend: site.baseurl }}){: .flex .flex--center}](https://www.surveymonkey.com/r/3P58K98)

Author: the acronyms look correct to me.  @atribolet are you thinking we just add a section at the end of the post with this list and definitions?

> [![Take the Survey]({{ "/images/SGID_2019_take_the_poll.png" | prepend: site.baseurl }}){: .flex .flex--center}](https://www.surveymonkey.com/r/3P58K98)

Reviewer: probably [this kramdown abbr feature](https://kramdown.gettalong.org/quickref.html#abbreviations)

and examples...

https://github.com/agrc/gis.utah.gov/blob/master/_posts/2019-05-30-thoughts-from-teaching-google-and-amazon-new-skills.md#L88-L91

> [![Take the Survey]({{ "/images/SGID_2019_take_the_poll.png" | prepend: site.baseurl }}){: .flex .flex--center}](https://www.surveymonkey.com/r/3P58K98)

Author: okay, I can work on that tomorrow morning.  thanks!

> [![Take the Survey]({{ "/images/SGID_2019_take_the_poll.png" | prepend: site.baseurl }}){: .flex .flex--center}](https://www.surveymonkey.com/r/3P58K98)

Author: done.  that was way easier than I was expecting.    ie: Ctrl+C  then Ctrl+V
